goentri.com.dkim-helper.json

```diff
@@ -0,0 +1,21 @@
+{
+  "providerId": "goentri.com",
+  "providerName": "Entri",
+  "serviceId": "dkim-helper",
+  "serviceName": "DKIM Helper",
+  "version": 1,
+  "logoUrl": "https://cdn.goentri.com/logo.svg",
+  "description": "Allows user to easily set up dkim record using Entri",
+  "syncPubKeyDomain": "goentri.com",
+  "syncRedirectDomain": "api.goentri.com, goentri.com",
+  "variableDescription": "dkimHost is dkim host and dkimValue is dkim value.",
+  "sharedProviderName": true,
+  "records": [
+    {
+      "type": "TXT",
+      "host": "%dkimHost%",
```

Comment:
It would be better for the template to be specific, if possible.
This one allows for basically any TXT record.
Better: `%dkimhost%._domainkey`